app/actions.py

```diff
@@ -177,6 +177,8 @@ def cursorMove(self, rowDelta, colDelta):
 
   def getCursorMoveAndMark(self, rowDelta, colDelta, markRowDelta,
       markColDelta, selectionModeDelta):
+    if self.penCol + colDelta < 0: #Catch cursor at beginning of line
```

Comment:
Trivia: I've been trying for a style similar to the Chromium style guide, though I haven't noted that yet. In that style there is two spaces before an end of line comment, a space after the # and a period at the end.

Like:

if self.penCol + colDelta < 0:  # Catch cursor at beginning of line.